### PR TITLE
Melange v3 compatibility

### PR DESCRIPTION
**Pull request by anmonteiro**

### CODE REVIEW

File: jest/jest.ml [hunk 1]
```diff
@@ -232,7 +234,10 @@ module Runner (A : Asserter) = struct
 
     let testAll name inputs callback =
       inputs |> List.iter (fun input ->
-        let name = {j|$name - $input|j} in
+        let name =
+          let input: string = Obj.magic input in
```
Comment (jchavarri) [inline jest/jest.ml:238]: Could be an identity external. Every time I see Obj.magic I remember the story with the syringes and the nice neighborhoods by Monsieur Leroy. 😄

**Review by jchavarri — COMMENTED** [review]
thx

File: jest/jest.ml (hunk 1 above)
Comment (anmonteiro) [reply jest/jest.ml:238]: what's this story referring to?

Comment (jchavarri) [reply jest/jest.ml:238]: took a while to find it as the old maillist server is not working anymore: https://sympa.inria.fr/sympa/arc/caml-list/2005-07/msg00223.html

**Pull request merged**